### PR TITLE
chore: add volunteer matifanger

### PR DESCRIPTION
**Pull request by matifanger**

### CODE REVIEW

**Review by accodes21 — CHANGES_REQUESTED** [review]
Looks good but try to restrict topics to max 2-3.

**Comment by codewithvoid** [comment]
Thanks for volunteering @matifanger :)

File: data/users/MatiFanger.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Matias Fanger",
+  "bio": "Senior Dev",
+  "image": "https://pbs.twimg.com/profile_images/1597759088934395904/VVMDI893_400x400.jpg",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["nuxt", "vue", "typescript", "javascript", "python"]
```
Comment (RakeshSangem) [inline data/users/MatiFanger.json:8]: could you limit these topics to 3.

**Review by RakeshSangem — CHANGES_REQUESTED** [review]

File: data/users/MatiFanger.json (hunk 1 above)
Comment (matifanger) [reply data/users/MatiFanger.json]: Done!

File: data/users/MatiFanger.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Matias Fanger",
+  "bio": "Senior Dev",
+  "image": "https://pbs.twimg.com/profile_images/1597759088934395904/VVMDI893_400x400.jpg",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["javascript", "python", "C/C#/C++"]
```
Comment (codewithvoid) [inline data/users/MatiFanger.json:8]: pick 1 of C/C#/C++

**Pull request closed without merging**